src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -975,7 +975,7 @@
 
 helps['network lb rule'] = """
     type: group
-    short-summary: Gather information, such as utilization, to be evaluated by rules
+    short-summary: Commands to manage load balancing rules based on information such as utilization.
```

Comment:
I'd vote to remove the `...based on information such as utilization.` part. At first read, it kinds of makes it sound more confusing that it should (at least to me), as opposed to just stating that the group allows managing the rules that dictate how load balancing works.